Makefile

```diff
@@ -238,7 +222,7 @@ package_src:
 	rm -Rf flexdll-$(VERSION)
 	mkdir flexdll-$(VERSION)
 	mkdir flexdll-$(VERSION)/test
-	cp -a *.ml Makefile $(COMMON_FILES) version.rc flexdll-$(VERSION)/
+	cp -a $(filter-out Compat.ml,$(wildcard *.ml)) Makefile $(COMMON_FILES) version.rc flexdll-$(VERSION)/
```

Comment:
I noticed when preparing the tags the other day that the inclusion of version.ml in the sources tarball is not explicit - it gets picked up if there's a build already there). Perhaps `package_src` should depend on `version.ml`? The wildcard has also occasionally included some debugging files (`bla.ml` and `bug.ml` :smile:)